dashboard/src/components/jobs/parameters/CompareJobParams.vue

```diff
@@ -43,7 +42,27 @@
           </label>
           <br />
         </div>
-        <div class="my-1">
+        <div v-if="containsPredicted">
+          <p>What is the time resolution of your data?</p>
+          <div class="ml-1 mt-1">
+            <input
+              type="radio"
+              id="hourly-resolution"
+              v-model="timeResolution"
+              value="leHourly"
```

Comment:
I wouldn't mind leaving this greyed and disabled since form changes are still required